docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -14,20 +14,20 @@ description: A software-defined asset is a description of how to compute the con
   <a href="https://www.youtube.com/watch?v=lRwpcyd6w8k" target="new">
     demo
   </a>{" "}
-  videos to get a quick look at Software-defined assets.
+  videos to get a quick look at asset definitions.
 </Note>
 
-An **asset** is an object in persistent storage, such as a table, file, or persisted machine learning model. A **software-defined asset** is a description, in code, of an asset that should exist and how to produce and update that asset.
+An **asset** is an object in persistent storage, such as a table, file, or persisted machine learning model. A **asset definition** is a description, in code, of an asset that should exist and how to produce and update that asset.
```

Comment:
I think the final version of this is 

"A **asset definition** is a description, in code, of an asset that should exist."

And then we *add* how to produce and update it in the case where we add a materialization function. This would make "external assets" just asset definitions for assets that are materialized elsewhere.

Reply:
Agree - the story behind this PR is that I started on the PR that makes that change, and thought it would be more reviewable if we made the "-> asset definition" change first.

Reply:
Awesome. Love where this is heading.